NitroxPatcher/Patches/Dynamic/GoTo_Patch.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class GoTo_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((GotoConsoleCommand gotoConsoleCommand) => gotoConsoleCommand.GotoPosition(default, default));
```

Comment:
Minor, just bc other patched have it this way:
```suggestion
    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((GotoConsoleCommand t) => t.GotoPosition(default, default));
```